index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="welcome-section">
+      <h1>My portfolio</h1>
+    </section>
+
+    <section id="projects">
+      <h1 class="project-tile">My project</h1>
+      <div>
+        <a id="profile-link" href="https://github.com/wassimchakib/html_css_dami_wassim"
+      target = "_blank"> 
+        <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" >
```

Comment:
- [ ] Good Job giving suffiscient details and reason to your review, However, I suggest to use suggestion instead of orders like recommended in ["How to give a code review"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#:~:text=Instead%20of%20giving%20orders%2C%20make%20suggestions.%20%22I%20think%20it%20would%20be%20better%20to%20use%20semantic%20tags%22%20sounds%20nicer%20than%20%22Use%20semantic%20tags%2C%22%20doesn%27t%20it%3F) :recycle: